.pytool/Plugin/UncrustifyCheck/UncrustifyCheck.py

```diff
@@ -153,6 +153,7 @@ def RunBuildPlugin(self, package_rel_path: str, edk2_path: Edk2Path, package_con
         """
         try:
             # Initialize plugin and check pre-requisites.
+            self._env = environment_config

```

Comment:
I don't see this being used anywhere

Reply:
It's used for this call `if self._env.GetValue("UNCRUSTIFY_IN_PLACE", "FALSE") == "TRUE":`.

Reply:
I'm blind. Thanks.